pallets/moonbeam-lazy-migrations/src/lib.rs

```diff
@@ -106,5 +111,36 @@ pub mod pallet {
 			)
 			.into())
 		}
+
+		// TODO(alexandru): This extrinsic should be removed once Gov V1 is removed.
+		// Note: We don't need to unreserve any funds, as they are assumed to be already
+		// unreserved prior to this operation and the proposal submission disabled.
+		#[pallet::call_index(1)]
+		#[pallet::weight(
+			<T as frame_system::Config>::DbWeight::get()
+				.reads_writes((*limit + 1).into(), (*limit).into())
+		)]
+		pub fn unlock_democracy_funds(
+			origin: OriginFor<T>,
+			limit: u32,
+		) -> DispatchResultWithPostInfo {
+			ensure_signed(origin)?;
+			ensure!(limit != 0, "Limit cannot be zero!");
+
+			// Unlock staked funds and remove the voting entry. This way we can keep track of what
+			// is left without extra cost.
+			let unlocked_accounts = VotingOf::<T>::iter()
```

Comment:
You should replace `VotingOf`by a raw key that you will construct from the pallet name and the storage item name,